### ISSUE TITLE
Add information from WPF VisualTreeHelper

### ISSUE THREAD

**Issue by Roemer** [Roemer/FlaUI#28]
In WPF, there is the VisualTreeHelper which allows you to browse the whole visual tree and get information out of it.
It is possible to attach to an application and get the visual tree out with information like text styles.
See https://wpfinspector.codeplex.com or https://snoopwpf.codeplex.com for examples.
We should add this as well to provide additional information for WPF applications.

**Comment by ChristianMoser** [comment]
Hi Roman,

I had the same Idea a while ago with a project spike called "XAML Inspector" which allows you to connect to a running, or lauch an new WPF application and then remote-control it through an agent.dll that gets loaded into the app and communicate through HTTP.  This could now merge into a new XAML-provider of FlaUI that could be written for UWP, WPF, WP10.

Regards
Christian

**Comment by Roemer** [comment]
This is a different approach than the one from WPFInspector right? But I like the idea having a small agent so that not the whole FlaUI needs to be injected. Do you have some code already?

**Comment by ChristianMoser** [comment]
Yes. WPF Inspector injects the whole application including the UI with all it's problems (style inheritance,...). The idea of XAML inspector was to have a small agent, that is easy to port to any XAML based platform that streams the data through a web-socket connection to the UI (or the FlaUI provider) that allows you to remote-debug or control an application.

**Comment by Roemer** [comment]
@ChristianMoser Did you start something already for the agent? Is there maybe some experimental code around?
